protractor-typescript-cucumber/Config/config.ts

```diff
@@ -0,0 +1,41 @@
+import {ProtractorBrowser} from 'protractor';
+/*
+The config folder includes all the configuration files
+This example config file displays the basic protractor-cucumber framework configuration
+ts-node compiler is needed for cucumberjs
+tags option for specific scenarios added
+**/
+export let config = {
+
+    directConnect: true,
+
+    baseUrl: 'http://juliemr.github.io/protractor-demo/',
```

Comment:
You can change the url to https://angular.github.io/protractor-cookbook/ng1/calculator/  . This way the test app used is part of the protractor cookbook. It is the same app.